runner.go

```diff
@@ -21,8 +22,9 @@ func createRun() {
 }
 
 func makeParams() *runParams {
+	slicedTags := strings.Split(tags, ",")
```

Comment:
Will this work if you have tags separated by a space and comma? `kind, of, like, this`